ansible_collections/arista/avd/roles/eos_commands_collect/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# tasks file for eos_commands_collect
+
+- name: Create output directory for each EOS device
+  file:
+    path: "{{ commands_backup_dir }}/{{ inventory_hostname }}"
```

Comment:
Should we delegate this task to `arista.avd.build_output_folders` to be compliant with all other roles from the collection ?

Reply:
Done!